tradingstrategy/binance/candlestick.py

```diff
@@ -0,0 +1,221 @@
+"""Get candlestick price and volume data from Binance.
```

Comment:
Add a docstring example code how to manually use this e.g. to download ETH-USD data.